### PR TITLE
qa/suites/tasks/cbt.py: Deprecating cosbench from Teuthology in preparation for deletion of cosbench from CBT

### PR DESCRIPTION
**Pull request by lee-j-sanders**
This is my first PR so please be gentle with me :-) 

[component]: qa/tasks

Cosbench support is being deleted from the Ceph Benchmarking Tool (CBT) (github.com/ceph/cbt.git) in favour of using HSBench and Elbencho.
Cosbench tests using CBT were implemented in Mimic, Nautilus,Octopus  and Pacific releases, but was then removed in  Quincy.
Pacific and older releases have now been Archived and no longer in support.
Later on we will add teuthology tests to add object tests to teuthology once the CBT work has been developed.

The python code in this PR is dormant infrastructure code to enable use of cosbench tests if a CBT test plan YAML was specified in the qa/suites/xxxx/ directories however there isn't any CBT test plan YAMLs enabling this function. Therefore the code to use cosbench in CBT is dormant and can safely be deleted.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

jenkins retest this please
jenkins test classic perf
jenkins test crimson perf
jenkins test signed
jenkins test make check
jenkins test make check arm64
jenkins test submodules
jenkins test dashboard
jenkins test dashboard cephadm
jenkins test api
jenkins test docs
jenkins render docs
jenkins test ceph-volume all
jenkins test ceph-volume tox
jenkins test windows
jenkins test rook e2e

</details>

### CODE REVIEW

**Comment by lee-j-sanders** [comment]
I have run

"ninja check"
Pushed my wip-ljs-rmcosbench branch to ceph-ci and this passed

image

Run the following teuthology tests: (TBD)

perf-basic
crimson-rados/perf
rados/perf

Here are the log outputs:

TBD.

**Comment by lee-j-sanders** [comment]
jenkins test make check

**Comment by lee-j-sanders** [comment]
jenkins test make check arm64

**Comment by lee-j-sanders** [comment]
jenkins test make check arm64

**Comment by lee-j-sanders** [comment]
I'm still waiting for the teuthology tests to complete at the moment. Submitted perf-basic rados/perf and crimson-rados/perf tests.

**Comment by lee-j-sanders** [comment]
@neha-ojha please can you review my PR thanks! I'm still waiting for teuthology tests to complete.

File: qa/README [hunk 1]
```diff
@@ -83,3 +83,8 @@ supported_distros as distros$ will be run just once: either on centos, rhel or
 ubuntu, chosen randomly.
 
 The teuthology code can be found in https://github.com/ceph/teuthology.git
+
+Note: The performance suites clone CBT from master here: https://github.com/ceph/cbt.git
+CBT will not support cosbench beyond release tag v0.3, therefore no qa suite should use cosbench.
```
Comment (ronen-fr) [inline qa/README:88]: QA?

**Comment by lee-j-sanders** [comment]
Pass on rados/perf QA suite:
https://pulpito.ceph.com/ljsanders-2024-10-10_09:53:50-rados:perf-wip-ljs-rmcosbench-distro-default-smithi/
image

Pass on perf-basic QA suite:
https://pulpito.ceph.com/ljsanders-2024-10-10_09:49:30-perf-basic-wip-ljs-rmcosbench-distro-default-smithi/

**Comment by lee-j-sanders** [comment]
reviewed-by: @ronen-fr

**Comment by batrick** [comment]
@lee-j-sanders please update your git config's user.email to the email you want to use with the project.

**Pull request merged**